htdocs/takepos/invoice.php

```diff
@@ -152,19 +152,19 @@ function fail($message)
 if ($invoiceid > 0) {
 	$ret = $invoice->fetch($invoiceid);
 } else {
-	$ret = $invoice->fetch('', '(PROV-POS'.$_SESSION["takeposterminal"].'-'.$place.')');
+	$ret = $invoice->fetch('', '(PROV-POS'. isset($_SESSION["takeposterminal"]) ? $_SESSION["takeposterminal"] : '' .'-'.$place.')');
```

Comment:
Who is executed first ? 
The  .  or the ?
Put parenthesis to avoid ambigous case

Reply:
Done